<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelSpecialElements</class>
 <widget class="QWidget" name="PanelSpecialElements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="groupBox_22">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Complete list</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_29">
        <item row="0" column="0">
         <widget class="PanelSuperList" name="panelSuperList" native="true">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,1">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Picture:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="WidgetPicture" name="widgetPicture" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="WidgetTilesetSettings" name="widgetTilesetSettings" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelSuperList</class>
   <extends>QWidget</extends>
   <header>panelsuperlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTilesetSettings</class>
   <extends>QWidget</extends>
   <header>widgettilesetsettings.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
